### ISSUE TITLE
End of Turn

### ISSUE THREAD

**Issue by Tgitoche** [jwatzman/murderisinthecards#1]
It seems easy to miss the idea that you need to end your turn.

**Comment by jwatzman** [comment]
Yeah, indeed, it is :\ Not really sure how to deal with this though -- as I understand the traditional rules, you can make an accusation at any point during your turn, including at the very end. So there needs to be basically some way to say "I am not going to make an accusation, the next person can go".

If you have any ideas, I'm really curious to hear them -- I haven't been able to come up with anything.

**Comment by Feuer-Paddl** [comment]
Hello Sir,

i hope you can help me. Terminal gives me this message "yarn install
00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'install'"

Thank you
